helm/camel-k/values.yaml

```diff
@@ -28,6 +28,7 @@ operator:
   resources: {}
   securityContext: {}
   tolerations: []
+  nodeSelector: {}
```

Comment:
This should be added to the documentation here: https://github.com/apache/camel-k/blob/a98dec1f8f260c2e478543c1ed3f19113c0cc5d2/helm/camel-k/README.md?plain=1#L107

Reply:
Done!